src/redux/actions/userActions.js

```diff
@@ -42,6 +43,13 @@ export const reduxShowReg = (value) => (dispatch) => {
 };
 /** stores the user data when a user logs in */
 export const reduxLogin = (user) => (dispatch) => {
+  const { user_info } = user || {};
```

Comment:
Note - we also will pass back is_guest from users.info

Reply:
ooooh yes I saw. Its good, but I found that its far tooo deep inside the user object. I just wanted to make it so that if we wanted to know that a user is a guest, we would just do `user.isGuest`. The code reads better that way 🤣 🤣